fvm/src/machine/mod.rs

```diff
@@ -199,7 +200,7 @@ pub struct NetworkContext {
     pub epoch: ChainEpoch,
 
     /// The UNIX timestamp (in seconds) of the current tipset
-    pub timestamp: u64,
+    pub timestamp: i64,
```

Comment:
Look... maybe we make a time machine? I'm happy to revert, this just made me twitch a bit.

Reply:
dunno, u64 seems fine. But it doesnt matter much either!

Reply:
actually, it is u64 at the source (tipset) so i suggest we keep it as is.